servers/onlyoffice-docspace/server.yaml

```
@@ -29,12 +29,12 @@ config:
       env: DOCSPACE_PASSWORD
       example: ditgor-p...
   env:
-    - name: DOCSPACE_BASE_URL
-      example: https://your-instance.onlyoffice.com
-      value: '{{onlyoffice-docspace.base_url}}'
-    - name: DOCSPACE_ORIGIN
-      example: https://your-instance.onlyoffice.com
-      value: '{{onlyoffice-docspace.origin}}'
+    - name: DOCSPACE_INTERNAL
+      example: false
+      value: '{{onlyoffice-docspace.internal}}'
+    - name: DOCSPACE_TRANSPORT
+      example: stdio
+      value: '{{onlyoffice-docspace.transport}}'
```

We currently only support stdio. Can you hardcode the value here and remove it from the configuration section?
```suggestion
    - name: DOCSPACE_TRANSPORT
      example: stdio
      value: 'stdio'
```